tests/impl.cpp

```diff
@@ -974,6 +980,18 @@ bool test_mm_load_sd(const double *p)
     return validateDouble(a, p[0], 0);
 }
 
+bool test_mm_load_pd(const double *p)
+{
+    __m128d a = _mm_load_pd(p);
+    return validateDouble(a, p[0], p[1]);
+}
+
+bool test_mm_loadu_pd(const double *p)
```

Comment:
I don't see that this function to be called in the tests

Reply:
Continuous work on #81